source/features/highlight-collaborators-and-own-discussions.tsx

```diff
@@ -11,8 +11,8 @@ async function highlightCollaborators(): Promise<false | void> {
 	}
 
 	const dom = await fetchDom(getRepoURL() + '/issues/show_menu_content?partial=issues/filters/authors_content');
-	const collaborators = select.all('.select-menu-item-text', dom).map(collaborator => {
-		return collaborator.firstChild!.textContent!.trim();
+	const collaborators = select.all('.SelectMenu-item [alt]', dom).map(collaborator => {
```

Comment:
You should cast it to `HTMLImageElement`:

```suggestion
	const collaborators = select.all<HTMLImageElement>('.SelectMenu-item [alt]', dom).map(collaborator => {
```

Reply:
@jerone thanks